lib/_debug_agent.js

```
@@ -102,11 +106,17 @@ function Client(agent, socket) {
   this.socket.on('close', function() {
     self.destroy();
   });
+  this.socket.on('error', function(err) {
+    // Silently handle socket errors
```

If we are silently ignoring the errors then why do we need this?